server/mustache-templates/newClient.mst

```diff
@@ -69,13 +69,49 @@
                         <small id="heartbeatPhoneNumbersHelp" class="form-text text-muted">Each phone number has +1 in front and separated by commas with no dashes or spaces, please (eg. +14445556789,+12223334444,+11231231234)</small>
                     </div>
                 </div>
-                <div class="form-group row justify-content-start row-no-gutters">
-                    <label for="incidentCategories" class="col-sm-3 col-form-label">Incident Categories:</label>
-                    <div class="col-sm-5">
-                        <input type="text" class="form-control" name="incidentCategories" value="{{incidentCategories}}" required>
-                        <small id="incidentCategoriesHelp" class="form-text text-muted">Each incident category is separated by commas with no quotes, please (eg. No One Inside,Person responded,Overdose,None of the above)</small>
-                    </div>
-                </div>
+<div class="form-group row justify-content-start row-no-gutters">
+    <label for="language" class="col-sm-3 col-form-label">Language:</label>
+    <div class="col-sm-5">
+        <select class="form-control" name="language" id="languageSelector" required>
+            <option value='en'>English</option>
+            <option value='es_us'>Spanish</option>
+            <option value='en'>English with Manual Input for Incident Categories</option>
+        </select>
+    </div>
+</div>
+
+<div class="form-group row justify-content-start row-no-gutters">
+    <label for="incidentCategories" class="col-sm-3 col-form-label">Incident Categories:</label>
+    <div class="col-sm-5">
+        <input type="text" class="form-control" name="incidentCategories" id="incidentCategoriesInput" required>
+        <small id="incidentCategoriesHelp" class="form-text text-muted">Each incident category is separated by commas with no quotes, please</small>
+    </div>
+</div>
+
+<script>
+    const englishIncidentCategories = 'No One Inside,Person responded,Overdose,None of the above'; 
+    const spanishIncidentCategories = 'Nadie adentro,La persona respondió,Sobredosis,Ninguna de las anteriores'; 
+
+    const languageSelector = document.getElementById('languageSelector');
+    const incidentCategoriesInput = document.getElementById('incidentCategoriesInput');
+
+    languageSelector.addEventListener('change', function() {
+        const selectedLanguage = this.value;
+
+        if (selectedLanguage === 'en') {
+            incidentCategoriesInput.value = englishIncidentCategories;
+        } else if (selectedLanguage === 'es_us') {
+            incidentCategoriesInput.value = spanishIncidentCategories;
+        } else {
+            incidentCategoriesInput.value = '';
+        }
+    });
+
+    // Trigger change event on page load to populate incident categories based on initial language selection
+    const event = new Event('change');
+    languageSelector.dispatchEvent(event);
+</script>
```

Comment:
The incident categories are automatically populated for both English and Spanish. This is extremally handy especially for the case of Spanish selection. it saves us from the hassle of manually typing them each time. 

The incident categories for the Spanish selection matches what were provided in the translation document and it appears these would be the only cases to use.

Reply:
Thanks Ahmad, I agree that this definitely is a valuable feature but my question is in regards to how these incident category values are stored in the incident_categories table in the database. Right now all the values in the incident_categories table are in english, so I'm wondering if this implementation will mean that they will be stored in Spanish?

Reply:
the values will be stored as they are entered, preserving their original format for both English and Spanish situations. they will be distinct entries and it is not guaranteed a uniform structure of incident categories since clients might have varied preferences.

Reply:
Thanks Ahmad, we talked a bit about this in our devices meeting (I've tagged you in a slack message with more details) and were thinking how the incident categories are stored in the database might be a broader conversation to have. If we start adding all the incident categories in different languages, it will impact our reporting and make our data harder to understand. Because of this, it may be better to keep them in english for now, before we have that bigger conversation with the team.